src/components/partnership/partnership.service.ts

```diff
@@ -397,12 +364,6 @@ export class PartnershipService {
       .call(matchUserPermissions, 'Partnership', id)
       .call(addAllSecureProperties, ...props)
       .optionalMatch([
-        node('canReadProject', 'Permission', {
-          property: 'project',
-          read: true,
-          active: true,
-        }),
-        relation('out', '', 'baseNode'),
```

Comment:
Was this check doing nothing?

Reply:
Yes, I don't think we need to check the `project` permission because while creating `Partnership`, we don't add `project' permission.